src/lib/data_mgr/SecureDataManager.cpp

```diff
@@ -263,6 +270,7 @@ bool SecureDataManager::login(const ByteString& passphrase, const ByteString& en
 
 	if (!RFC4880::PBEDeriveKey(passphrase, salt, &pbeKey))
 	{
+		sleep(Configuration::i()->getInt("login.fail_delay", 0));
```

Comment:
On Windows I would expect this to be a capitalized `Sleep` (https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleep) which takes in milliseconds, instead of the POSIX `sleep` which takes seconds, so I'm not sure how this could work for both platforms.